cookbooks/bcpc_kafka/recipes/kafka.rb

```diff
@@ -71,8 +71,10 @@
 include_recipe 'bcpc_kafka::default'
 include_recipe 'kafka::default'
 
+node.force_default[:kafka][:ulimit_file] = 32_768
```

Comment:
`node.force_default['kafka']['ulimit_file']`, please.

Yes, symbols vs strings is my nitpick complaint this time; apologies all around.